### ISSUE TITLE
can't build on osx or linux

### ISSUE THREAD

**Issue by ahoward** [klevernothings/libopkele#1]
this is what i'm getting on osx

cfp:~/src/git > git clone git://github.com/hacker/libopkele.git && cd libopkele && autoconf && ./configure
Initialized empty Git repository in /Users/ahoward/src/git/libopkele/.git/
remote: Counting objects: 1322, done.
remote: Compressing objects: 100% (364/364), done.
remote: Total 1322 (delta 972), reused 1278 (delta 948)
Receiving objects: 100% (1322/1322), 213.90 KiB | 239 KiB/s, done.
Resolving deltas: 100% (972/972), done.
configure.ac:6: error: possibly undefined macro: AM_INIT_AUTOMAKE
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
configure.ac:11: error: possibly undefined macro: AC_PROG_LIBTOOL
configure.ac:29: error: possibly undefined macro: AC_SUBST
configure.ac:34: error: possibly undefined macro: AC_CHECK_SHAREDPTR
configure.ac:51: error: possibly undefined macro: AC_MSG_ERROR
configure.ac:72: error: possibly undefined macro: AC_DEFINE
configure.ac:131: error: possibly undefined macro: AC_WITH_DOXYGEN
configure.ac:132: error: possibly undefined macro: AC_WITH_DOT
configure.ac:134: error: possibly undefined macro: AM_CONDITIONAL

and on linux

[ahoward@vs5865 libopkele]$ git clone git://github.com/hacker/libopkele.git && cd libopkele && autoconf && ./configure
Initialized empty Git repository in /home/ahoward/libopkele/libopkele/.git/
remote: Counting objects: 1322, done.
remote: Compressing objects: 100% (364/364), done.
remote: Total 1322 (delta 972), reused 1278 (delta 948)
Receiving objects: 100% (1322/1322), 213.90 KiB, done.
Resolving deltas: 100% (972/972), done.
configure.ac:6: error: possibly undefined macro: AM_INIT_AUTOMAKE
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
configure.ac:11: error: possibly undefined macro: AC_PROG_LIBTOOL
configure.ac:29: error: possibly undefined macro: AC_SUBST
configure.ac:34: error: possibly undefined macro: AC_CHECK_SHAREDPTR
configure.ac:51: error: possibly undefined macro: AC_MSG_ERROR
configure.ac:72: error: possibly undefined macro: AC_DEFINE
configure.ac:131: error: possibly undefined macro: AC_WITH_DOXYGEN
configure.ac:132: error: possibly undefined macro: AC_WITH_DOT
configure.ac:134: error: possibly undefined macro: AM_CONDITIONAL

am i being stupid - or is this a bug?

**Comment by hacker** [comment]
Let's start by trying `sh autogen.sh` instead of `autoconf && ./configure` ?

**Comment by ahoward** [comment]
not even close ;-)  this worked however:

http://gist.github.com/101709

i have the same issue and solution with mod_auth_openid on both linux and osx with latest autotools version.

**Comment by hacker** [comment]
Well, linking aclocal.d to acinclude.d is wrong, but may work. Actually, the directory should be created by libtoolize, which is invoked by autogen.sh before the rest of auto-stuff.

I've just committed and merged my old changes, that I made for mac os x. I've just tested this - `sh autogen.sh PKG_CONFIG_PATH=/sw/lib/pkgconfig/ && make` on my macbook with the next branch and it seems to work fine... On linux it has always worked, provided you use autogen.sh
